packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -33,6 +33,13 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
         const value = this.valueForCell(x, y);
         return this.displayString(value, column.constituentType, column.name);
       }
+      if (
+        row.hasChildren &&
+        row.depth <= x + 1 &&
+        this.table.groupedColumns.includes(column)
```

Comment:
This is unlikely to cause performance issues in this particular scenario since there's not too many cells that will realistically fall into this block, but right now this call will need to iterate through the `groupedColumns` array looking for `column` each time you get in here. If they had thousands of grouped columns (which, realistically is extremely unlikely, but we need to be prepared for anyway), this could cause performance issues.
We should be caching the `groupedColumns` array as a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) and using `.has` on that instead.
See `getCachedFilterableColumnSet` for an example, but just make something like `getCachedGroupedColumnSet` that returns the memoized column groups and call .has on that.

Also - to be double paranoid, I would also get the value of the cell `const value = this.valueForCell(x, y)` and only return the empty string if it is indeed `null`. And include a comment in this block about why we are doing this (can reference the issue number if you want), as this is a strange thing to do without more context.